metricflow/cli/dbt_connectors/dbt_config_accessor.py

```diff
@@ -105,7 +105,8 @@ def build_semantic_manifest_from_dbt_project_root(project_root: Path) -> Semanti
             raise ModelCreationException(
                 f"Unable to find {full_path_to_manifest}\n"
                 "Please ensure that you are running `mf` in the root directory of a dbt project "
-                "and that the semantic_manifest JSON exists."
+                "and that the semantic_manifest JSON exists. If this is your first time running "
+                "`mf`, run `dbt parse` to generate the semantic_manifest JSON."
```

Comment:
dbt n00b question: do you think people might get confused if they run `dbt parse` and then try a query and have it return stale (or even empty) results because they needed a `dbt build` or `dbt run` as well? If so it might be worth adding a bit more detail here.

Reply:
Make sense, thanks for the context! I think I'll merge this as-is and we can see if there's added confusion. It's too early to know whether people running this will be managing metrics only, or if they'll be more typically doing end to end model -> metrics development.

In theory the errors they get should be familiar to anyone who has tried to do a `dbt parse` and then run a query in their warehouse on top of the models.